specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
I admit to being confused.  It was my understanding that sampling always happens across an entire contest, independent of geography.  Thus, the type of this model concept only mentions the product of contests and sequences of ballot contest info.  I intended to filter off of this sequence based upon county.